app/assets/javascripts/components/stories/Sprints.js

```diff
@@ -10,18 +10,17 @@ const defaultProps = {
   sprints: [],
 };
 
-const renderSprints = (sprints, fetchStories) => {
-  return sprints.map(
-    (sprint, index) =>
+const renderSprints = (sprints, fetchStories) =>
+  sprints.map(
+    sprint =>
       sprint ? (
```

Comment:
There is a chance of a sprint to be null?
If so, I would filter first and then map them.
```js
const renderSprints = (sprints, fetchStories) =>
  sprints
    .filter(Boolean)
    .map(sprint => (
      ...
    ))
```

Reply:
I believe that we can use just:
```
sprints.map(
    sprint =>
      <Sprint
        key={sprint.number}
        sprint={sprint}
        fetchStories={fetchStories}
      />
  )
```
I did some tests and search in the code and has no cases when sprint is null. thanks!